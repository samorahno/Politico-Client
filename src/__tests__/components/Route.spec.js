import React from 'react';
import { shallow } from 'enzyme';
import Route from '../../routes';

describe('Route Component', () => {
    it('should render correctly in "debug" mode', () => {
        const component = shallow(<Route />);
        expect(component).toMatchSnapshot();
    });
});
