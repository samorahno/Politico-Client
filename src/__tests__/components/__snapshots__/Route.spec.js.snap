// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Route Component should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
